types/serverless/plugins/aws/provider/awsProvider.d.ts

```diff
@@ -792,6 +810,56 @@ declare namespace Aws {
     interface Credentials {
         [key: string]: any;
     }
+
+    interface Stages {
+        [key: string]: Stage | undefined;
+        default?: Stage | undefined;
+    }
+
+    interface Stage {
+        params?: { [key: string]: string } | undefined;
```

Comment:
Issue: I don't think this is right - `params` looks to be a property _next_ to `stages`, not within the object